### PR TITLE
Elastic heart like animation

### PR DESCRIPTION
**Pull request by cupcake08**
Hi, 
This is a heart like animation I made today. 
Hope you like it.
 I'm new to flutter animations. 
So, Any suggestions will be great.

### CODE REVIEW

File: heart_animation_2/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+This project is inspired from Dribble Design
+
+- [Like Animation](https://dribbble.com/shots/15890672-Like-Animation)
+
+Here is how it looks :
+
+<img src="demo/demo.gif" height=300>
+
+
+## Getting Started
```
Comment (hiashutoshsingh) [inline heart_animation_2/README.md]: I think we can remove these below lines just to keep it clean.

File: heart_animation_2/lib/animation.dart [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      lowerBound: 0.0,
+      upperBound: 0.1,
+    );
+
+    _animation = CurvedAnimation(
+      parent: _controller,
+      curve: const ElasticOutCurve(0.8),
+    );
+
+    super.initState();
```
Comment (hiashutoshsingh) [inline heart_animation_2/lib/animation.dart]: Let's have this super.initState() line as the first line.
[Read here why](https://stackoverflow.com/questions/58400001/should-i-call-super-initstate-at-the-end-or-at-the-beginning/58400324#58400324)

File: heart_animation_2/lib/extensions.dart [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import 'dart:developer' as devtools show log;
+
+extension Log on Object {
+  /// just for debugging purposes
+  void log() => devtools.log(toString());
+}
```
Comment (hiashutoshsingh) [inline heart_animation_2/lib/extensions.dart:6]: Not related to this file, let's delete [home.dart](https://github.com/hiashutoshsingh/flutter_heart_challenge/pull/7/files#diff-11a1406e0b19a26aace7ac992689a58a22aa482f308eb4aae525832230f00650) file as there is no use of it.

**Review by hiashutoshsingh — CHANGES_REQUESTED** [review]
Hey @cupcake08 👋🏼 
Thank you for raising the PR, I tried the demo and it look good.

Just couple of changes nothing big, please go through it and lmk if you have questions and after that I think we are good to go. 👍🏼

File: heart_animation_2/lib/animation.dart (hunk 2 above)
Comment (cupcake08) [reply heart_animation_2/lib/animation.dart]: Yeah sure.
Thanks for the StackOverflow link.

File: heart_animation_2/README.md (hunk 1 above)
Comment (cupcake08) [reply heart_animation_2/README.md]: I made the given changes

File: heart_animation_2/lib/extensions.dart (hunk 3 above)
Comment (cupcake08) [reply heart_animation_2/lib/extensions.dart:6]: deleted home.dart file

**Review by hiashutoshsingh — APPROVED** [review]
Everything looks great now and thanks for your contribution. 🚀

**Pull request merged**